.github/workflows/tests.yml

```diff
@@ -0,0 +1,33 @@
+name: Tests
+
+# More conservative about duplicate tests due to tests accessing real files
+on: [pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        python-version: [3.9]
```

Comment:
The publish-release.yml is using 3.8. Seems like we should use the same versions, or rather maybe the tests should run against multiple versions at least one of which matches the version used in publish-release.yml.